llvm/test/Transforms/LoopVectorize/AArch64/partial-reduce-dot-product-mixed.ll

```diff
@@ -1,6 +1,6 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --check-globals none --filter-out-after "^scalar.ph:" --version 4
-; RUN: opt -passes=loop-vectorize -enable-epilogue-vectorization=false -mattr=+i8mm,+dotprod -S < %s | FileCheck %s
-; RUN: opt -passes=loop-vectorize -enable-epilogue-vectorization=false -mattr=+dotprod -S < %s | FileCheck %s --check-prefix=CHECK-NOI8MM
+; RUN: opt -passes=loop-vectorize -enable-epilogue-vectorization=false -mattr=+i8mm,+dotprod --vectorizer-maximize-bandwidth -S < %s | FileCheck %s
+; RUN: opt -passes=loop-vectorize -enable-epilogue-vectorization=false -mattr=+dotprod -S --vectorizer-maximize-bandwidth < %s | FileCheck %s --check-prefix=CHECK-NOI8MM
```

Comment:
Is this necessary? If not, can you remove it?

Reply:
It is, since otherwise we don't test the scalable pathway as the max VF available is vscale x 4.

Reply:
Sorry for the unfortunate timing, but I've just come to realise that this flag is probably required if you want the SVE sudot/usdot tests to enforce scalable vectors. Otherwise, it would still pick a fixed-length VF for the `+i8mm` case, where you actually want to test this specifically uses a scalable VF. Could you remove the last commit (486aa4ca815823000d9422b447947f2cabf4b50c) ?

Reply:
Done.